pipelines/h3c/rtp/v1/rtp.toml

```diff
@@ -8,11 +8,20 @@ base_cpu = 1
 batch_system = "slurm"
 
 [WorkFlow]
-actions = ["SETUP", "EXTRACT_HH", "UPLOAD_HH", "ADD_RTP_PROCESS_RECORD", "TEARDOWN"]
+actions = ["SETUP", "EXTRACT_HH", "UPLOAD_HH", "ANT_METRICS", "ADD_MC_ANT_METRICS", "ADD_LIBRARIAN_ANT_METRICS", "ADD_RTP_PROCESS_RECORD", "TEARDOWN"]
```

Comment:
Toml lets us break lists over lines, that will make this much more readable than one mega long line

Reply:
Make it three lines!